frontend/src/pages/LogsPage/SearchForm/SearchForm.tsx

```diff
@@ -180,13 +183,6 @@ const Search: React.FC<{
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [initialQuery])
 
-	useEffect(() => {
```

Comment:
Re-use of the Search component on the log alert page does not work well if we always have the form and combobox state in sync (since we send a request to fetch the histogram based on the form state)